ksqldb-functional-tests/src/main/java/io/confluent/ksql/test/tools/TopicInfoCache.java

```diff
@@ -87,6 +88,13 @@
       )
   );
 
+  // Internal topic names that are ignored by the TopicInfoCache::get() method
+  private static final Set<Pattern> IGNORE_INTERNAL_TOPIC_NAMES_PATTERNS = ImmutableSet.of(
+      Pattern.compile(TOPIC_PATTERN_PREFIX + ".*-FK-JOIN-SUBSCRIPTION-REGISTRATION-\\d+-topic"),
```

Comment:
Can we add a quick comment into the code explaining why we're ignoring these internal topics for now, and that it's only temporary (until we have a long-term fix)? Ideally we'd file the information in a Github issue and simply put the link to the Github issue in the comment. Whichever you prefer.

Reply:
Done.